app/validators/answers/option_answer_validator.py

```diff
@@ -1,6 +1,9 @@
 from functools import cached_property
 
 from app.validators.answers import AnswerValidator
+from app.validators.rules.rule_validator import RulesValidator
+
+MIN_OPTIONS_BY_ANSWER_TYPE = {"Checkbox": 1, "Radio": 2, "Dropdown": 2}
```

Comment:
If these strings are referenced often, I think it'd be better to create and use constants

Reply:
I could make an `AnswersTypes` class and reference the types using that, but not sure if that refactor should be part of this. As it would change references in a lot of unrelated files. Throughts?

Reply:
Isn't this new code though? I can't see where it moved from so assumed it was new

Reply:
Yes, that dict is new, but I assume you are referring to having some const to refer to the answer types?. What I mean is, the answer types such as "Checkbox" is referenced in other places that I haven't touched, so I'm not sure if you are suggesting refactor for this file only?

Reply:
Added an `AnswerType` Enum class and updated the references in the new code.